mirai-core-api/src/commonMain/kotlin/message/data/MarketFace.kt

```diff
@@ -28,13 +29,49 @@ public interface MarketFace : HummerMessage {
      */
     @MiraiExperimentalApi
     public val id: Int
-
     override val key: MessageKey<MarketFace> get() = Key
 
     override fun contentToString(): String = name
 
     public companion object Key :
         AbstractPolymorphicMessageKey<HummerMessage, MarketFace>(HummerMessage, { it.safeCast() }) {
         public const val SERIAL_NAME: String = "MarketFace"
+
+        /**
+         * 生成指定点数的骰子表情.
+         *
+         * [value] 需要生成的骰子点数，默认为随机点数
+         */
+        @JvmName("dice")
+        @JvmStatic
+        @JvmOverloads
+        public fun dice(value: Int = Random.nextInt(1, 7)): MarketFace {
+            require(value in 1..6) {
+                "Dice value must in 1 to 6"
+            }
+            return Dice(value)
+        }
     }
+}
+
+/**
+ * 用于标识可发送的商城表情
+ */
+public interface MarketFaceTemplate
+
+/**
+ * 骰子表情
+ *
+ * 生成请参见 [MarketFace.Key.dice]
+ */
+public class Dice internal constructor(
+    /**
+     * 骰子点数
+     */
+    public val value: Int
+) : MarketFace, MarketFaceTemplate {
+    override val id: Int get() = 11464
+    override fun toString(): String = "[mirai:marketface:$id,$name]"
+
+    override val name: String get() = "[骰子:$value]"
 }
```

Comment:
添加 `init{}` 对 `value` 进行检查

```suggestion
/**
 * 骰子表情
 *
 * 生成请参见 [MarketFace.Key.dice]
 */
public class Dice internal constructor(
    /**
     * 骰子点数
     */
    public val value: Int
) : MarketFace, MarketFaceTemplate {
    override val id: Int get() = 11464
    override fun toString(): String = "[mirai:marketface:$id,$name]"

    override val name: String get() = "[骰子:$value]"
    init {
        require(value in 1..6) {
            "Dice value must in 1 to 6"
        }
    }
}
```